Makefile

```diff
@@ -52,6 +52,13 @@ format:
 	docker-compose exec web black .
 
 format-check:
+	docker-compose exec web isort -rc -tc --atomic --diff .
 	docker-compose exec web black --diff .
 
 setup: setup_env build
+
+super: 
```

Comment:
Can we `superuser` this instead? Super is ambiguous.

Reply:
Sure. Will do.